src/Commands/TankDrive.cpp

```diff
@@ -2,10 +2,8 @@
 #include "TankDrive.h"
 #include "../Subsystems/Chassis.h"
 
-using namespace robovikes;
-
 TankDrive::TankDrive(bool automaticShifting) :
-	mAutomaticShifting(automaticShifting) {
+	mAutomaticShifting(true) {
```

Comment:
all these weird indents are cause by GitHub/Smartgit registering a tab as 4 spaces when on are computer it shows two. I will have to change preferences to replace tabs with actual spaces